eng/common/scripts/job-matrix/Create-PrJobMatrix.ps1

```diff
@@ -205,7 +205,7 @@ function GeneratePRMatrixForBatch {
         }
 
         if ($batchSuffixNecessary) {
-          $outputItem["name"] = $outputItem["name"]  + "_$batchNamePrefix$batchCounter"
```

Comment:
I saw this when it snuck in and I didn't want to pay another eng/common round trip when I did. Finally addressing it 😆